lidar_apollo_cnn_seg_detect/README.md

```diff
@@ -24,6 +24,17 @@ $ make distribute
 
 **Recompile Autoware to build the node.**
 
+## The Pretrained model
+
+Use this link to download the pretrained model from Baidu:
+
+https://github.com/ApolloAuto/apollo/tree/master/modules/perception/production/data/perception/lidar/models/cnnseg
```

Comment:
Would you mind making this a link to a specific commit's version of the model rather than `master`? `master` will continue to change which will affect the model which is available at this link.

Reply:
Sure

Reply:
done